lib/github-pages-health-check/domain.rb

```diff
@@ -63,28 +63,27 @@ class Domain < Checkable
         "43.249.72.133",
         "43.249.73.133",
         "43.249.74.133",
-        "43.249.75.133"
+        "43.249.75.133",
+
+        # 2018 Move to GitHub assigned IP space
+        "192.30.252.153",
```

Comment:
Users will begin to receive deprecation emails when they build, and the status in their repository settings will start notifying them of outdated IPs.